src/components/sidebar/FloatingActionButton.jsx

```diff
@@ -15,9 +15,7 @@ const FloatingActionButton = props => (
     onClick={props.onClick}
     onKeyDown={props.onClick}
   >
-    <div className='fab__line fab__line--1' />
-    <div className='fab__line fab__line--2' />
-    <div className='fab__line fab__line--3' />
+    +
```

Comment:
Instead of using the 'plus' character, you could use two divs. They need to have same height and width, and one of the divs should be rotates 180 degrees. :)